### PR TITLE
Feat/ROE-2664: Retrieve application data from DB for the `beneficial-owner-individual` page

### PR DESCRIPTION
**Pull request by mwejuli-ch**
### JIRA link

https://companieshouse.atlassian.net/browse/ROE-2664

### Change description

- Retrieves application data from the API for the beneficial-owner-individual` page/screen
- Modifies tests to match new/updated functionality
- Includes minor formatting changes

### Work checklist

- [x] Tests added where applicable
- [x] UI changes meet accessibility criteria

### CODE REVIEW

**Review by markpit — APPROVED** [review]

File: src/utils/beneficial.owner.individual.ts [hunk 1]
```diff
@@ -1,8 +1,18 @@
 import { NextFunction, Request, Response } from "express";
 import { Session } from "@companieshouse/node-session-handler";
+import { addCeasedDateToTemplateOptions } from "../utils/update/ceased_date_util";
+import { saveAndContinue } from "../utils/save.and.continue";
+import { logger } from "../utils/logger";
+import { v4 as uuidv4 } from "uuid";
```
Comment (lduranteau) [inline src/utils/beneficial.owner.individual.ts:6]: it is just a small detail, but is it possible to keep the library imports at the top and then the local imports?

What do you think?

File: test/controllers/update/update.beneficial.owner.individual.controller.spec.ts [hunk 2]
```diff
@@ -16,24 +16,31 @@ import { NextFunction, Request, Response } from "express";
...
 import app from "../../../src/app";
+
 import { authentication } from "../../../src/middleware/authentication.middleware";
 import { companyAuthentication } from "../../../src/middleware/company.authentication.middleware";
+import { ErrorMessages } from '../../../src/validation/error.messages';
+import { ServiceAddressKey, ServiceAddressKeys } from "../../../src/model/address.model";
+import { ApplicationDataType } from '../../../src/model';
+import { hasUpdatePresenter } from "../../../src/middleware/navigation/update/has.presenter.middleware";
+import * as config from "../../../src/config";
+import { DateTime } from "luxon";
```
Comment (lduranteau) [inline test/controllers/update/update.beneficial.owner.individual.controller.spec.ts:27]: same as above

File: src/utils/beneficial.owner.individual.ts (hunk 1 above)
Comment (mwejuli-ch) [reply src/utils/beneficial.owner.individual.ts]: agreed. resolved now.

File: test/controllers/update/update.beneficial.owner.individual.controller.spec.ts (hunk 2 above)
Comment (mwejuli-ch) [reply test/controllers/update/update.beneficial.owner.individual.controller.spec.ts]: also resolved.

**Review by lduranteau — APPROVED** [review]

**Pull request merged**